src/api-reference/financial-integration/v4.financial-documents-schema.markdown

```diff
@@ -28,311 +28,312 @@ layout: reference
+`lineItemDescription`|-|-|-
+`lineItemPurchaseOrderNumber`|String|-|The purchase order number associated with the invoice line item (for a multiple purchase order-based invoice), or the purchase order number associated with the header (for a single purchase order-based invoice). Maximum length 48 characters.
+`lineItemQuantity`|-|-|-
+`lineItemSequenceOrder`|Numeric|Integer| Line item number for the line item related to this Journal Entry.
+`lineItemUnitPrice`|-|-|-
+`poLineNumber`|Numeric|-|The PO line item number associated to the payment request. Maximum length 10 characters.
+`receiptNumber`|Numeric|-|The Goods Receipt number.
+`receiptQuantity`|Numeric|-|The Goods Received quantity.
+`receiptItemID`|Numeric|-|The ID of the Goods Receipt tied to the Invoice and PO Line.
 
 ### <a name="scinvlisa"></a>Invoice - Line Item Ship To Address
 
-| **Name** | **Type**  | **Format** | **Description** |
-|------------|-------------|-------------|------------|
-| `address1` | String  | - | Ship To Address Line 1. |
-| `address2` | String  | - | Ship To Address Line 2. |
-| `address3` | String  | - | Ship To Address Line 3. |
-|`addressCode` | String  | - | Ship To Address Code value. |
-| `addressCreateDate` | Date | - | Date the Ship To Address was created. |
-| `addressID` | String  | - | The Primary Key value for the Ship To address. |
-| `addressName` | String  | - | Address Name. |
-| `city` | String  | - | City defined on the Ship To Address. |
-| `contactFirstName` | String  | - | Ship To Contact First Name.|
-| `contactLastName` | String  | - | Ship To Contact Last Name. |
-| `countryCode` | String  | - | ISO Country Code. |
-| `phoneNumber` | Numeric | Integer | Phone Number on address. |
-| `postalCode`  | String  | - | Postal Code value.  |
-| `state` | String  | - | State value on Ship To Address. |
-| `taxJurisdictionCode` | String  | - | The Tax Jurisdiction Code value defined on the Ship To Address. |
+Name|Type|Format|Description
+---|---|---|---
+`address1`|String|-|Ship To Address Line 1.
+`address2`|String|-|Ship To Address Line 2.
+`address3`|String|-|Ship To Address Line 3.
+`addressCode`|String|-|Ship To Address Code value.
+`addressCreateDate`|Date|-|Date the Ship To Address was created.
+`addressID`|String|-|The Primary Key value for the Ship To address.
+`addressName`|String|-|Address Name.
+`city`|String|-|City defined on the Ship To Address.
+`contactFirstName`|String|-|Ship To Contact First Name.
+`contactLastName`|String|-|Ship To Contact Last Name.
+`countryCode`|String|-|ISO Country Code.
+`phoneNumber`|Numeric| Integer| Phone Number on address.
+`postalCode`|String|-|Postal Code value.
+`state`|String|-|State value on Ship To Address.
+`taxJurisdictionCode`| String|-|The Tax JurisdictionCode value defined on the Ship To Address.
+`poLineNumber`|Numeric|-|The PO line item number associated to the payment request. Maximum length 10 characters.
```

Comment:
@RoAnnC1 Can you verify if `poLineNumber` should be here? Thanks.

Reply:
The Invoice fields are still under design so we'll be getting an updated schema for the Invoice content by end of January. For now we can publish as-is, knowing we'll update it later.